pdr_backend/pdr_dashboard/test/test_plots.py

```diff
@@ -154,9 +154,9 @@ def test_get_figures_and_metrics(
     assert fig_profit.layout["title"] == "Profit"
     assert fig_costs.layout["title"] == "Costs"
 
-    assert fig_accuracy.layout["yaxis_title"] == "'%' accuracy over time"
-    assert fig_profit.layout["yaxis_title"] == "OCEAN profit over time"
-    assert fig_costs.layout["yaxis_title"] == "Stake (OCEAN) at a time"
+    assert fig_accuracy.layout["yaxis"]["title"] == "Accuracy(%)"
```

Comment:
why do we need to re-set these? Aren't they part of create_figure?

Reply:
probably to check that create_figure sets the values in the right place